package.json

```diff
@@ -43,7 +47,8 @@
     "gulp-zip": "^4.2.0",
     "jquery": "^3.3.1",
     "nunjucks": "^3.1.3",
-    "sass-lint": "^1.12.1"
+    "sass-lint": "^1.12.1",
+    "start-server-and-test": "^1.7.11"
```

Comment:
Don't really like the idea of having another dependency that just starts the server. Can you investigate this further?

`"start-server": "npm start"` is "start": "gulp",